roles/splunk_common/tasks/main.yml

```diff
@@ -122,14 +122,9 @@
 - include_tasks: enable_dsp.yml
   when: "'dsp' in splunk and 'enable' in splunk.dsp and splunk.dsp.enable"
 
-- include_tasks: start_splunk.yml
-
-- include_tasks: set_certificate_prefix.yml
```

Comment:
also this playbook?

Reply:
This playbook, set_certificate_prefix I essentially incorporated into roles/splunk_common/tasks/enable_splunkd_ssl.yml. If we turn on ssl we might as well set the certificate prefix in the same file instead of having two different steps.